src/server/bnetserver/Realms/WorldListener.cpp

```diff
@@ -65,7 +68,7 @@ void WorldListener::HandleOpen()
 {
     try
     {
-        _worldSocket->bind(std::string("tcp://*:") + std::to_string(_worldListenPort));
+        _worldSocket->bind(std::string("tcp://"+_bindIp+":") + std::to_string(_worldListenPort));
```

Comment:
Can you explain this change? ZMQ sockets are used for ipc, they have nothing to do with clients